labelbox/schema/export_filters.py

```diff
@@ -45,19 +45,33 @@ class DatasetExportFilters(SharedExportFilters):
 
 
 def validate_datetime(string_date: str) -> bool:
-    """helper function validate that datetime is as follows: YYYY-MM-DD for the export"""
+    """helper function to validate that datetime's format: "YYYY-MM-DD" or "YYYY-MM-DD hh:mm:ss" 
+    or ISO 8061 format "YYYY-MM-DDThh:mm:ss±hhmm" (Example: "2023-05-23T14:30:00+0530")"""
     if string_date:
-        for fmt in ("%Y-%m-%d", "%Y-%m-%d %H:%M:%S"):
+        for fmt in ("%Y-%m-%d", "%Y-%m-%d %H:%M:%S", ISO_8061_FORMAT):
             try:
                 datetime.strptime(string_date, fmt)
                 return True
             except ValueError:
                 pass
         raise ValueError(f"""Incorrect format for: {string_date}.
-        Format must be \"YYYY-MM-DD\" or \"YYYY-MM-DD hh:mm:ss\"""")
+        Format must be \"YYYY-MM-DD\" or \"YYYY-MM-DD hh:mm:ss\" or ISO 8061 format \"YYYY-MM-DDThh:mm:ss±hhmm\""""
+                        )
     return True
 
 
+def covert_to_utc_if_iso8061(string_date: str, timezone_str: Optional[str]):
+    """helper function to convert datetime to UTC if it is in ISO_8061_FORMAT and set timezone to UTC"""
+    try:
+        date_obj = datetime.strptime(string_date, ISO_8061_FORMAT)
+        date_obj_utc = date_obj.replace(tzinfo=timezone.utc)
+        string_date = date_obj_utc.strftime(ISO_8061_FORMAT)
+        timezone_str = "UTC"
+    except ValueError:
```

Comment:
For my own understanding, if the date is not in ISO8601 format, will it fail at the first line, `date_obj = datetime.strptime(string_date, ISO_8061_FORMAT)`, and therefore `timezone_str` will not be set to `UTC`?

Reply:
yes, that is correct. It will throw a ValueError which we are catching.